Gemfile

```diff
@@ -5,32 +5,34 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
-gem 'rails', '~> 5.1.5'
+gem 'coffee-rails', '~> 4.2'
+gem 'devise'
+gem 'jbuilder', '~> 2.5'
```

Comment:
This one can go

Reply:
To be clear, you want me to remove which gem? Not sure I want to remove 'coffe-rails' since it broke everything last time, remember?

Reply:
I mean `jbuilder`